cmd/agent/openstack_iid_attestor/main.go

```diff
@@ -27,16 +26,12 @@ import (
 // IIDAttestorPlugin implements the nodeattestor Plugin interface
 type IIDAttestorPlugin struct {
 	logger   hclog.Logger
-	config   *IIDAttestorPluginConfig
 	metaData *openstack.Metadata
```

Comment:
the `metaData` is not used now, remove it?

Reply:
fix: 0fe4d7e